services/collectd/config.go

```diff
@@ -7,19 +7,29 @@ import (
 )
 
 const (
+	// DefaultBindAddress is the default port to bind to
 	DefaultBindAddress = ":25826"
 
+	// DefaultDatabase is the default DB to write to
 	DefaultDatabase = "collectd"
 
+	// DefaultRetentionPolicy is the default retention policy of the writes
 	DefaultRetentionPolicy = ""
 
-	DefaultBatchSize = 1000
+	// DefaultBatchSize is the default write batch size.
+	DefaultBatchSize = 5000
 
-	DefaultBatchPending = 5
+	// DefaultBatchPending is the default number of pending write batches.
+	DefaultBatchPending = 10
 
+	// DefaultBatchTimeout is the default batch timeout.
 	DefaultBatchDuration = toml.Duration(10 * time.Second)
 
 	DefaultTypesDB = "/usr/share/collectd/types.db"
+
+	// DefaultUDPReadBuffer is the default UDP read buffer
+	// increasing this increases the number of UDP packets that can be handled.
```

Comment:
This is a little vague. Why not be precise and repeat the text in the Go docs?

"Sets the size of the operating system's receive buffer associated with the UDP traffic "